src/azure-cli/azure/cli/command_modules/acr/_docker_utils.py

```diff
@@ -217,18 +217,22 @@ def _get_credentials(cmd,  # pylint: disable=too-many-statements
     url = 'https://' + login_server + '/v2/'
     try:
         challenge = requests.get(url, verify=(not should_disable_connection_verify()))
-        if challenge.status_code in [403]:
-            raise CLIError("Looks like you don't have access to registry '{}'. ".format(login_server) +
-                           "To see configured firewall rules, run" +
-                           " 'az acr show --query networkRuleSet --name {}'. ".format(registry_name) +
-                           "Details: https://docs.microsoft.com/azure/container-registry/container-registry-health-error-reference#connectivity_forbidden_error")  # pylint: disable=line-too-long
+        if challenge.status_code == 403:
+            raise CLIError("Looks like you don't have access to registry '{}'. "
+                           "To see configured firewall rules, run "
+                           "'az acr show --query networkRuleSet --name {}'. "
+                           "Details: https://aka.ms/acr/errors#connectivity_forbidden_error"
+                           .format(login_server, registry_name))  # pylint: disable=line-too-long
```

Comment:
Details: -> Please refer to?

Reply:
> Do you still need the pylint disable?

You're right. Since we've shortened the URL, we can remove this.

> Details: -> Please refer to?

I think we can change it to "Please refer to https://aka.ms/acr/errors#connectivity_forbidden_error for more information." from "Details: https://aka.ms/acr/errors#connectivity_forbidden_error".